packages/devtools_app/lib/src/info/info_controller.dart

```diff
@@ -18,6 +18,19 @@ class InfoController extends DisposableController
     _listenForFlutterVersionChanges();
   }
 
+  /// Return the FlutterVersion for the current connected app, or `null`
+  /// if the `flutterVersion` service extension does not exist.
+  static Future<FlutterVersion> getFlutterVersion() async {
+    final app = serviceManager.connectedApp;
+    if (!(await app.isFlutterApp)) {
```

Comment:
You may be able to use isFlutterAppNow depending on when this is called

Reply:
Gotcha; I'm not sure we know yet whether `isFlutterAppNow` has been populated or not at this location.